config.sh

```diff
@@ -16,12 +16,29 @@ function build_wheel {
     fi
```

Comment:
In the line above, change `build_osx_wheel $@` to `wrap_wheel_builder build_osx_wheel $@`

Reply:
Changed, thank you!